assets/templates/signup.html

```diff
@@ -30,48 +30,36 @@
+  <main class="panel-wrapper">
+    <article class="main-panel">
+      <header>
         <img src="img/logo-doppler.svg" class="logo-doppler" alt="Doppler">
-        <p class="head-signin">¿Ya tienes una cuenta? <a href="#" class="link-green" target="_blank">INGRESA</a></p>
-      </div>
+        <p class="p-signin">¿Ya tienes una cuenta? <a href="#" class="link-green" target="_blank">INGRESA</a></p>
+      </header>
+      <h5>REGÍSTRATE</h5>
+      <p class="p-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p>
 
-      <div class="main-panel__content">
-        <h5><strong>REGÍSTRATE</strong></h5>
-        <p>Crea una cuenta GRATIS hasta 500 Suscriptores.</p>
-      </div>
-
-      <div class="main-panel__rgpd">
+      <div class="p-legal">
         <p>Doppler te informa que los datos de carácter personal que nos proporciones al rellenar el presente formulario serán tratados por Making Sense LLC (Doppler) como responsable de esta web.</p>
         <p><strong>Finalidad:</strong> Darte de alta en nuestra plataforma y brindarte los servicios que nos requieras.</p>
         <p><strong>Legitimación:</strong> Consentimiento del interesado.</p>
         <p><strong>Destinatarios:</strong> Tus datos serán guardados por Doppler, Zoho como CRM, Digital Ocean, Cogeco Peer1 y Rackspace como empresas de hosting.</p>
-        <p><strong>Información adicional:</strong> En la <a href="#" class="link-green" target="_blank">Política de Privacidad</a> de Doppler encontrarás información adicional sobre la recopilación y el uso de su información personal por parte de
-          Doppler,
-          incluida información
-          sobre acceso,
-          conservación, rectificación, eliminación, seguridad, transferencias transfronterizas y otros temas.</p>
-      </div>
-      <hr class="line-dashed">
-      <div class="main-panel__promotion">
-        <p>Pst! si tienes un código promocional podrás ingresarlo al confirmar tu cuenta. <a href="#" class="link-green" target="_blank">HELP</a></p>
+        <p><strong>Información adicional:</strong> En la <a href="#" class="link-green" target="_blank">Política de Privacidad</a> de Doppler encontrarás información adicional sobre la recopilación y el uso de su información personal por
+          parte de Doppler, incluida información sobre acceso, conservación, rectificación, eliminación, seguridad, transferencias transfronterizas y otros temas.</p>
       </div>
-      <div class="main-panel__footer">
+      <p class="p-promotion">Pst! si tienes un código promocional podrás ingresarlo al confirmar tu cuenta. <a href="#" class="link-green" target="_blank">HELP</a></p>
+      <footer>
         <p>© 2019 Doppler LLC. Todos los derechos reservados.</p>
-        </footer>
-    </section>
-
+      </footer>
+    </article>
     <section class="feature-panel">
-      <div class="feature-content">
-        <p><strong>EDITOR DE EMAILS</strong></p>
+      <article class="feature-content">
+        <h6>EDITOR DE EMAILS</h6>
```

Comment:
What do you think about making uppercase part of the style and write normally?

```suggestion
        <h6>Editor de emails</h6>
```

Reply:
I think use property text-transform: uppercase is not necessary in this case, because the text is short and can replace for text in uppercase.

Reply:
Mmm, creo que no es un tema de facilidad o comodidad. Si la mayusculas se usan como forma de estilar, me parece que lo correcto es que se haga a través de estilos. Por ejemplo de esa forma se aseguraría uniformidad en otros casos.

Tal vez la cuestión práctica tenga más importancia de la que me imagino, en ese caso no prestes atención a mi comentario.

Reply:
I think it also varies in what position we take to develop this type of cases, @GusBaamonde layers also this would be something that we would have to define. If all the texts that are in uppercase have to be transformed by scss or we put them in capital letters in the text.